src/index.html

```diff
@@ -7,8 +7,50 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link 
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" 
+    rel="stylesheet" type="text/css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="navigationBar">
+      <a href="#home" class="nav_link">
+        <img src="images/logo.png" alt="icon of the website">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item nav_itemApple"> 
```

Comment:
```suggestion
          <li class="nav_item"> 
```
Why do you need this class? You don't use it